vitkon/sky-todos
# Assessment

[![Build Status](https://travis-ci.org/vitkon/sky-todos.svg?branch=feature%2Fassessment)](https://travis-ci.org/vitkon/sky-todos) [![Coverage Status](https://coveralls.io/repos/github/vitkon/sky-todos/badge.svg?branch=feature%2Fassessment)](https://coveralls.io/github/vitkon/sky-todos?branch=feature%2Fassessment)

#### What's this PR do?
- adds the ability to delete items
- adds the ability to toggle todo items
- adds a filter that can be toggled to hide completed items

#### Where should the reviewer start?
App.js

#### How should this be manually tested?
- start the app `npm start`
- mark one item as completed
- toggle the filter to hide completed items and make sure that completed items are no longer visible
- toggle the filter to show back completed items and make sure that completed items are now listed with the rest of todos
- delete one of the items and make sure it is not displayed anymore
